app/presenters/history_event.rb

```diff
@@ -0,0 +1,47 @@
+require "delegate"
+
+# HistoryEvent wraps & delegates to C2Version
```

Comment:
we haven't generally been doing class descriptions in C2. We can discuss if we want to move in that direction, but as of right now it's not something we're doing.